src/Symfony/Component/Validator/Tests/Constraints/CallbackValidatorTest.php

```diff
@@ -227,4 +226,28 @@ public function testAnnotationInvocationMultiValued()
 
         $this->assertEquals(new Callback(array(__CLASS__.'_Class', 'validateCallback')), $constraint);
     }
+
+    public function testPayload()
```

Comment:
Imo this should be changed to something like `testPayloadIsPasswordToCallback` to reflect what we actually expect to happen in the test.